packages/mdc-textfield/README.md

```diff
@@ -206,24 +246,24 @@ UX for client-side form field validation.
          placeholder="Full-Width Textfield"
          aria-label="Full-Width Textfield">
 </div>
-<div class="mdc-textfield mdc-textfield--multiline mdc-textfield--fullwidth">
-  <textarea class="mdc-textfield__input"
-            placeholder="Full-Width multiline textfield"
-            rows="8" cols="40"
-            aria-label="Full-Width multiline textfield"></textarea>
+
+<div class="mdc-textfield mdc-textfield--fullwidth mdc-textfield--textarea">
+  <textarea id="full-width-textarea" class="mdc-textfield__input" rows="8"></textarea>
+  <label for="full-width-textarea" class="mdc-textfield__label">Textarea Label</label>
 </div>
 ```
 
 Note that **full-width text fields do not support floating labels**. Labels should not be
-included as part of the DOM structure for full-width text fields.
+included as part of the DOM structure for full-width text fields. Full-width textareas
+behave normally.
 
 ### Text Field Boxes
 
 ```html
 <div class="mdc-textfield mdc-textfield--box">
   <input type="text" id="tf-box" class="mdc-textfield__input">
-  <label for="tf-box" class="mdc-textfield__label">Your Name</label>
   <div class="mdc-textfield__bottom-line"></div>
+  <label for="tf-box" class="mdc-textfield__label">Your Name</label>
```

Comment:
Why was this moved below bottom-line? It seems to be above it in our demos.

Reply:
Doesn't matter, but will change for the sake of alignment